app/experimenter/visualization/api/v3/views.py

```diff
@@ -41,6 +41,30 @@ class Statistic:
 STATISTICS_FOLDER = "statistics"
 METADATA_FOLDER = "metadata"
 
+GROUPED_METRICS = [
+    {
+        "name": "search_metrics",
+        "metrics": [
+            "searches_with_ads",
+            "search_count",
+            "organic_search_count",
+            "tagged_follow_on_search_count",
+            "tagged_search_count",
+        ],
+    },
+    {
+        "name": "usage_metrics",
+        "metrics": [
+            "uri_count",
+            "active_hours",
+        ],
+    },
```

Comment:
you know, I think we could do one more group here called "guardrails". It would include `days_of_use`, `retained`, and `search_count`.